lib/types/rules/stylistic-issues.d.ts

```diff
@@ -312,6 +322,7 @@ export interface StylisticIssues extends Linter.RulesRecord {
      * Rule to require or disallow spacing between function identifiers and their invocations.
      *
      * @since 3.3.0
+     * @deprecated since 8.53.0, please use the [corresponding rule](https://eslint.style/rules/js/func-call-spacing) in `@stylistic/eslint-plugin-js`.
```

Comment:
`@eslint/stylistic` uses a different name:

```suggestion
     * @deprecated since 8.53.0, please use the [corresponding rule](https://eslint.style/rules/js/function-call-spacing) in `@stylistic/eslint-plugin-js`.
```